robyn/processpool.py

```diff
@@ -13,6 +14,24 @@
 from robyn.ws import WS
 
 
+def is_port_in_use(port: int) -> bool:
+    async def echo(websocket):
+        async for message in websocket:
+            await websocket.send(message)
+
+    async def main():
+        server = await websockets.serve(echo, "localhost", port)
+        server.close()
+
+    try:
+        asyncio.run(main())
+        return False
+    except OSError:
+        return True
+    except Exception:
+        return True
```

Comment:
Hey @Mr-Sunglasses :wave: 

One suggestion - let us return errors here. It is a recommended way to 
 not do a generic catch all for the Exception